### PR TITLE
Feat/add issue reporting template

### PR DESCRIPTION
**Pull request by galin-iliev**

### CODE REVIEW

File: io-connect-desktop-components/src/components/ChannelSelector/ChannelSelector.tsx [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-    >
-      <ThemeProvider>
-        <ChannelSelector />
-      </ThemeProvider>
-    </GlueProvider>
-  );
-}
-export default ChannelSelectorWrapper;
+import { ChannelSelector, ThemeProvider } from "@interopio/components-react";
+import { GlueProvider } from "@glue42/react-hooks";
```
Comment (arjunah) [inline io-connect-desktop-components/src/components/ChannelSelector/ChannelSelector.tsx:2]: We have @interopio/react-hooks now

File: io-connect-desktop-components/src/components/ChannelSelector/ChannelSelector.tsx [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+import Glue from "@glue42/desktop";
```
Comment (arjunah) [inline io-connect-desktop-components/src/components/ChannelSelector/ChannelSelector.tsx:3]: @interopio/desktop

File: io-connect-desktop-components/package.json [hunk 3]
```diff
@@ -6,7 +6,6 @@
   "dependencies": {
     "@glue42/desktop": "^5.23.1",
```
Comment (arjunah) [inline io-connect-desktop-components/package.json:7]: @interopio/desktop and @interopio/react-hooks

**Review by arjunah — CHANGES_REQUESTED** [review]

**Comment by galin-iliev** [comment]
The changes from this pull request are in the following PR as well:
https://github.com/InteropIO/templates/pull/11
I\m closing this one.

**Pull request closed without merging**